### PR TITLE
Update cagg material for TimescaleDB 2.7

### PR DESCRIPTION
**Pull request by charislam**
# Description
Updates cagg material for new final form of caggs in TimescaleDB 2.7.

# Review checklists
Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

- [ ] Is the content technically accurate?
- [ ] Is the content complete?
- [ ] Is the content presented in a logical order?
- [ ] Does the content use appropriate names for features and products?
- [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

- [x] Is the content free from typos?
- [x] Does the content use plain English?
- [x] Does the content contain clear sections for concepts, tasks, and references?
- [x] Are procedure and highlight tags used appropriately?
- [x] Has the index been updated appropriately?
- [x] Have any images been uploaded to the correct location, and are resolvable?
- [x] Are all links provided in reference style, and resolvable?
- [x] Have you checked the built version of this content?

### CODE REVIEW

File: api/create_materialized_view.md [hunk 1]
```diff
@@ -81,6 +81,7 @@ Optional `WITH` clause options:
 |-|-|-|-|
 |`timescaledb.materialized_only`|BOOLEAN|Return only materialized data when querying the continuous aggregate view|`FALSE`|
 |`timescaledb.create_group_indexes`|BOOLEAN|Create indexes on the continuous aggregate for columns in its `GROUP BY` clause. Indexes are in the form `(<GROUP_BY_COLUMN>, time_bucket)`|`TRUE`|
+|`timescaledb.finalized`|BOOLEAN|In TimescaleDB 2.7 and above, use the new version of continuous aggregates, which stores finalized results for aggregate functions. Supports non-parallelizable aggregate functions, `FILTER` and `JOIN` clauses, and the `DISTINCT` keyword.|`TRUE`|
```
Comment (charislam) [inline api/create_materialized_view.md:84]: Does it actually support JOINs? I couldn't tell from the release/PR notes.

File: timescaledb/how-to-guides/continuous-aggregates/about-continuous-aggregates.md [hunk 2]
```diff
@@ -51,20 +51,35 @@ hasn't been aggregated yet. This gives you up-to-date results on every query. If
...
+The following table summarizes aggregate function support in continuous
+aggregates:
+
+|Function, clause, or feature|TimescaleDB 2.7 and above|Below TimescaleDB 2.7|
+|-|-|-|
+|Parallelizable aggregate functions|✅|✅|
+|Non-parallelizable aggregate functions|✅|❌|
+|Ordered-set aggregates|✅|❌|
+|Hypothetical-set aggregates|✅|❌|
+|Window functions|✅|❌|
```
Comment (charislam) [inline timescaledb/how-to-guides/continuous-aggregates/about-continuous-aggregates.md:75]: Not sure about this?

File: timescaledb/how-to-guides/continuous-aggregates/about-continuous-aggregates.md [hunk 3]
```diff
@@ -51,20 +51,35 @@ hasn't been aggregated yet. This gives you up-to-date results on every query. If
...
+|`DISTINCT`|✅|❌|
+|`FILTER`|✅|❌|
+|`JOIN`|✅|❌|
```
Comment (charislam) [inline timescaledb/how-to-guides/continuous-aggregates/about-continuous-aggregates.md:78]: Also not sure about this?

**Comment by charislam** [comment]
Hey @NunoFilipeSantos, I identified all the places in the docs that need to change for finalized caggs and wrote up the skeleton, but need an SME review as I'm not 100% sure on what all the new compatibilities are! Other suggestions also are welcome if there are other side effects of this change

File: api/create_materialized_view.md (hunk 1 above)
Comment (fabriziomello) [inline api/create_materialized_view.md:84]:
```suggestion
|`timescaledb.finalized`|BOOLEAN|In TimescaleDB 2.7 and above, use the new version of continuous aggregates, which stores finalized results for aggregate functions. Supports non-parallelizable aggregate functions, `FILTER` and `ORDER BY`, and the `DISTINCT` clause in aggregate functions.|`TRUE`|
```

Seems the `JOIN` is wrong in this context... see my suggestion above.

File: timescaledb/how-to-guides/continuous-aggregates/about-continuous-aggregates.md (hunk 2 above)
Comment (fabriziomello) [inline timescaledb/how-to-guides/continuous-aggregates/about-continuous-aggregates.md:75]:
```suggestion
```

We're not supporting Window Functions yet in Caggs

File: api/create_materialized_view.md (hunk 1 above)
Comment (fabriziomello) [reply api/create_materialized_view.md:84]: Just to clarify we don't support `JOIN` in Continuous Aggregate definition yet.

**Review by fabriziomello — APPROVED** [review]
LGTM

**Review by Loquacity — APPROVED** [review]

**Pull request merged**